app/assets/stylesheets/views/_covid.scss

```diff
@@ -80,13 +72,27 @@
   }
 }
 
+.covid__logo-wrapper {
+  margin-bottom: govuk-spacing(7);
+
+  @include govuk-media-query($from: tablet) {
+    margin-bottom: 0;
+    position: relative;
+    height: 108px;
```

Comment:
Why 108px? I think it's because that's the height of the text, but this needs to take into account different text sizes, e.g.

<img width="1037" alt="Screenshot 2020-04-24 at 14 52 38" src="https://user-images.githubusercontent.com/861310/80220167-5aa24d80-863b-11ea-992e-5c0c91fd266a.png">

If we're absolutely positioning the logo, have you tried making the parent of the whole block relative and putting the logo bottom right with a max width of one third?

Reply:
I'll try a few things and let you know

Reply:
The new solution is much better but might need tweaking to fully simulate the grid columns, as there's a small amount of overlap.

<img width="704" alt="Screenshot 2020-04-27 at 11 21 04" src="https://user-images.githubusercontent.com/861310/80361693-48155780-8879-11ea-860b-67bff0358bbb.png">